Configurations/README

```diff
@@ -45,19 +45,24 @@ In each table entry, the following keys are significant:
                            Note: if the same feature is both enabled
                            and disabled, disable wins.
 
+        as              => The Assembler compiler.  This is not always
```

Comment:
Wow! I've never thought of assembler as "Assembler compiler". Take `man as` for example. Assembler is effectively opposed to compiler... As in "compiler generates input for assembler"... If you try to pull analogy with C compiler, then word before compiler is supposed to refer to language, while assembler as term refers rather to program...

Reply:
Hmmm, wouldn't it be appropriate to list it as AS now? Along with CC, CPP, etc...

Reply:
Well, therein lies the double meaning, doesn't it?  "Assembler" refers to the language as well as the program that translates it into an object file.  Now, this gets even more complicated with MACRO32 on Alpha, where VAX assembler is translated (compiled) into Alpha assembler, and it really is refered to as an "assembler compiler".  But that's a rather unusual case, so I'll just remove "compiler"...

Reply:
That some C compilers translate C to assembler is quite irrelevant, actually.

Reply:
Fixed